dev/com.ibm.ws.grpc_fat/test-applications/StoreApp.war/src/com/ibm/testapp/g3store/servletStore/HelloServlet.java

```diff
@@ -0,0 +1,43 @@
+/*******************************************************************************
+ * (c) Copyright IBM Corporation 2017.
```

Comment:
2020

Reply:
A more basic question: do we actually need to have these `HelloServlet` classes checked in?